content/en/docs/KRE/user/60_kais_runner_sdk/_index.md

```diff
@@ -97,6 +98,49 @@ func handler(ctx *kre.HandlerContext, data *any.Any) (proto.Message, error) {
 }
 ```
 
+## How to provide node functions to kre
+
+Once the `init` and `handler` functions have been declared and implemented, they have to be served in a certain way, so runners recognize them.
+
+### For runners V2
+
+#### Python V2
+
+Just by having two functions named __init__ and __handler__. Python runners interpret what source code is given.
+
+#### Golang V2
+
+Golang nodes use the Golang runners as a library. To load them up, inside the `main` function in the `main.go` file the runner must be run passing down the functions that serve as `init` and `handler`.
+
+```go
+func main() {
+  kre.Start(handlerInit, handler)
+}
+```
+
+### For runners V3
+
+In this runner version you can decide which function implemented by the node will act as the `handler` depending on the source of the incoming payload.
+
+#### Python V3
+
+The function named _default_handler_ will act as the default handler. Custom handlers will be declared inside a dictionary named _custom_handlers_, the dictionary will have as key the source node name and as value the function to execute.
+
+#### Golang V3
+
+To load default and custom handlers, they have to be used in the function `Start` from the runner's library as their arguments. The first argument will be the _init_ function, the second the _default handler_ and the third a map of key the source node's name and value the function to execute.
+
+```go
+func main() {
+  handlers := map[string]kre.Handler{
+    "entrypoint": entrypointHandler,
+    "nodeA": nodeAHandler,
+  }
+
+  kre.Start(handlerInit, defaultHandler, handlers)
+}
+```
+
 ## Secrets & Environment
 
 ## Config
```

Comment:
It would interesting to add Python examples too here.

Reply:
Got it :+1: